core/templates/domain/exploration/read-only-exploration-backend-api.service.spec.ts

```diff
@@ -95,139 +72,150 @@ describe('Read only exploration backend API service', function() {
       version: 1,
       state_classifier_mapping: {}
     };
-  }));
+  });
 
-  afterEach(function() {
-    $httpBackend.verifyNoOutstandingExpectation();
-    $httpBackend.verifyNoOutstandingRequest();
+  afterEach(() => {
+    httpTestingController.verify();
   });
 
   it('should successfully fetch an existing exploration from the backend',
-    function() {
-      var successHandler = jasmine.createSpy('success');
-      var failHandler = jasmine.createSpy('fail');
+    fakeAsync(() => {
+      let successHandler = jasmine.createSpy('success');
+      let failHandler = jasmine.createSpy('fail');
 
-      $httpBackend.expect('GET', '/explorehandler/init/0').respond(
-        sampleDataResults);
-      ReadOnlyExplorationBackendApiService.fetchExploration(
+      readOnlyExplorationBackendApiService.fetchExploration(
         '0', null).then(successHandler, failHandler);
-      $httpBackend.flush();
+      var req = httpTestingController.expectOne(
```

Comment:
Please use `let` here and in other places in this file.

Reply:
Done